packages/restlessness-cli/rln-backend/src/endpoints/put-endpoints-by-id/index.ts

```diff
@@ -0,0 +1,6 @@
+import 'module-alias/register';
+import { LambdaHandler } from '@restlessness/core';
```

Comment:
I made an ad hoc handler for rln backend, give a look to the others index.ts ;-)

Reply:
@androsanta

Reply:
i didn't notice 👍